jenkins_finish_build.py

```diff
@@ -0,0 +1,42 @@
+# /// script
+# dependencies = [
+#   "valkey[libvalkey]",
+# ]
+# ///
```

Comment:
I've used here python scripts because it would be the easiest, but don't know how to deal with the dependency, can we get this pre-installed? Otherwise it would need to be converted to running `nc` commands directly, which seems like a nightmare.